examples/restaurant_demo/restaurant.py

```diff
@@ -1,19 +1,19 @@
+import math
+import statistics as s
+import pydp as dp  # our privacy library
+import pandas as pd
+from collections import defaultdict
 import sys  # isort:skip
 sys.path.append("../pydp")  # isort:skip
 
```

Comment:
I may have missed this somewhere Do you think having/adding a comprehensive README is/would be a good idea? Example: Restaurant - Differential Privacy by Google: 1) Go: https://github.com/google/differential-privacy/blob/main/examples/go/README.md; 2) Java: https://github.com/google/differential-privacy/blob/main/examples/java/README.md

Reply:
I believe that we will add a README later, just like the carrots demo! right @chinmayshah99 ?